### DISCUSSION TITLE
Sprint 5 Features

### DISCUSSION THREAD

**Discussion by seannkelleyy** [seannkelleyy/Mario-Remake#97] [Ideas]
Lets starting getting some Sprint 5 ideas!

**Comment by seannkelleyy** [comment]
Here are some ideas of mine:
- React level creator
     - Show coordinates on hover
- Power downs (constant jumping, decrease in size, lose coins etc.)
- Dev menu that lets you spawn enemies or change physics

**Comment by jliet359** [comment]
One that seems interesting to me is the idea of enemies having power ups!
Also one of the other groups had enemy AI, maybe that's also a possibility?

**Comment by Soul279** [comment]
I plan to had new enemies and power ups. Mainly bowser, kamek, implementing the bullets and piranahs, floating koopas, and lakitu. For power ups i wanted to give mario guns like for example a pistol, shotgun, rocketlaucher. If there is enough time to complete it i would also want to add an infinite level mode where the level scrolls and generates infinitly with a scaling difficulting becoming more difficult as you progress.

**Comment by hood276** [comment]
I think we should have a title screen with four options. Play level 1-1, create a new level, load a previously created level, or exit the game. I think we could also have controller support.

**Comment by DomArtman** [comment]
It would be cool to do an AI escort with like Toad or implement the Boo's with dead enemies.
